### DISCUSSION TITLE
Page preview when hovering over title

### DISCUSSION THREAD

**Discussion by Limezy** [athensresearch/athens#1039] [Feature Requests]
Sawhney from Discord community sent a link to a very interesting software : https://saga.so/
One of the things I liked is that when you hover over a link, it'll show you a preview of the page linked.
Would be nice to have this in Athens, don't you think ?

**Comment by rjbergerud** [comment]
Andy Matuschak also does this on his own personal note-taking system https://notes.andymatuschak.org/z4mAF1uBV96r72e4NjLcDaujEyTPGiUQJEj8C

I think it would be nice as a feature you could toggle

**Reply by jefftangx** [reply]
I really want this. @shanberg what's the status of dropdowns/popovers?

**Reply by shanberg** [reply]
> I really want this. @shanberg what's the status of dropdowns/popovers?

I realized the MaterialUI stuff we've already included in the project contains a ton of utilities to handle this, we just need to refactor a little. I've been trying on my own, but I bet it'd take 15 minutes for a more experienced CLJS dev.

**Reply by shanberg** [reply]
@tangjeff0 We have a working example already in the Presence view, so we have all we need. I'm just blowing a gasket trying to put my clojure brain back on to implement this more generally.

**Comment by jsmorabito** [comment]
just made #1159 which seems to be related. if they're not too dissimilar maybe they could be batched together by whoever takes up the challenge
